Source/ASVideoNode.mm

```diff
@@ -707,6 +710,13 @@ - (void)resetToPlaceholder
 
 #pragma mark - Playback observers
 
+- (void)applicationDidBecomeActive:(NSNotification *)notification
+{  
+  if (self.shouldBePlaying && self.isVisible) {
+    [self play];
```

Comment:
What if the video is paused when the user backgrounds the app? Should we save that state and only start playing on resume if the video wasn't paused?